releasenotes/notes/redact-commands-from-redis-resources-c1bc594b98f299c5.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    redis: Updates the resource name of redis spans to only include the command (ex: SET/GET/PUSH/....).
+    To revert to the previous behavior set ``DD_REDIS_RESOURCE_ONLY_COMMAND`` to ``FALSE``.
```

Comment:
```suggestion
    redis: Set the span resource only the command to avoid the potential for high cardinality if key arguments are included. To retain to the previous behavior which can include keys, set ``DD_REDIS_RESOURCE_ONLY_COMMAND=false``.
```